packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -163,6 +170,34 @@ export function RepositoryMixin<T extends Class<any>>(superClass: T) {
         }
       }
     }
+
+    /**
+     * Update or recreate the database schema for all repositories.
+     *
+     * **WARNING**: By default, `migrateSchema()` will attempt to preserve data
+     * while updating the schema in your target database, but this is not
+     * guaranteed to be safe.
+     *
+     * Please check the documentation for your specific connector(s) for
+     * a detailed breakdown of behaviors for automigrate!
+     *
+     * @param options Migration options, e.g. whether to update tables
+     * preserving data or rebuild everything from scratch.
+     */
+    async migrateSchema(options?: SchemaMigrationOptions): Promise<void> {
```

Comment:
I would prefer to have two methods and use `updateSchema` as default.

- migrateSchema(...)
- updateSchema(...)

Reply:
Or at least an array of repo names.